ogr/services/github/service.py

```diff
@@ -47,64 +56,49 @@ def __init__(
         github_app_id: str = None,
         github_app_private_key: str = None,
         github_app_private_key_path: str = None,
+        tokman_instance_url: str = None,
+        github_authentication: GithubAuthentication = None,
         **_,
     ):
+        """
+        If multiple authentication methods are provided, they are prioritised:
+            1. Tokman
+            2. GithubApp
+            3. Token (which is also default one, that works without specified token)
```

Comment:
Post-rename leftover:

```suggestion
            3. TokenAuthentication (which is also default one, that works without specified token)
```